source/sass/utility/_color.scss

```diff
@@ -17,6 +17,7 @@ $color-utils: (
             light:      (color: $color-light),
             lighter:    (color: $color-lighter),
             lightest:   (color: $color-lightest),
+            primary-contrasting: (color: $color-primary-contrasting),
```

Comment:
For the items above there should be equal spaces to have everything lined perfectly (styling)